mne/viz/topomap.py

```diff
@@ -2382,3 +2382,81 @@ def _plot_corrmap(data, subjs, indices, ch_type, ica, label, show, outlines,
     fig.canvas.draw()
     plt_show(show)
     return fig
+
+
+def _trigradient(x, y, z):
+    """Taking gradients of z on a mesh"""
+    from matplotlib.tri import Triangulation, CubicTriInterpolator
+    tri = Triangulation(x.ravel(), y.ravel())
+    tci = CubicTriInterpolator(tri, z.ravel())
+    dx, dy = tci.gradient(tri.x, tri.y)
+    return dx, dy
+
+def plot_arrowmap(data, info_from, info_to=None, scale=1e-10, axes=None):
+
+
+    """Plotting arrow map
```

Comment:
Parameters

is missing. See how other docstrings are formatted

Reply:
Done